.mvn/wrapper/maven-wrapper.properties

```diff
@@ -15,5 +15,5 @@
 # specific language governing permissions and limitations
 # under the License.
 
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
-wrapperUrl=https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/3.1.1/maven-wrapper-3.1.1.jar
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.0/apache-maven-3.9.0-bin.zip
```

Comment:
Just in case, [3.9.1](https://maven.apache.org/docs/3.9.1/release-notes.html) is newest patch release. Seems that it solves two connection related issues [MNG-7708](https://issues.apache.org/jira/browse/MNG-7708) and [MNG-7721](https://issues.apache.org/jira/browse/MNG-7721).

Reply:
OK. Let me try to upgrade here.